multiplatform/greeter/.gitignore

```diff
@@ -0,0 +1,42 @@
+# 🤖 Android Studio
+*.iml
+# We ignore everything from the IDE, except non user specific things
+/.idea/*
+# Auto-import exclusion list (Editor -> General -> Auto Import)
+!.idea/codeInsightSettings.xml
```

Comment:
minor: I usually just ignore the whole directory

Reply:
Yeah I can do that since I'm not really including any IDEA files anyway. This just comes from my personal template.